qemu/tests/cfg/x86_cpu_model.cfg

```diff
@@ -84,6 +84,16 @@
             no RHEL.9
             flags = "vme sse2 sse fxsr mmx clflush pse36 pat cmov mca pge mtrr sep apic cx8 mce pae msr tsc pse de  popcnt cx16 lm nx syscall misalignsse sse4a abm lahf_lm fpu"
             model_pattern = "AMD Opteron 23xx \(Gen 3 Class Opteron%s\)"
+        - SapphireRapids:
+            only HostCpuVendor.intel
+            required_qemu = [8.0.0-1,)
```

Comment:
Suggest `required_qemu = [8.0.0,)`

Reply:
> Suggest `required_qemu = [8.0.0,)`

@yanan-fu Hi, the Fixed Version is 'qemu-kvm-8.0.0-1.el9', I think we can keep the strict qemu version.

Reply:
You are totally right for downstream,  but not exactly for upstream. 
For new functions be introduced with qemu-kvm rebase,  it is better specify the main version only. 

Only for functions be introduced with a minor version, we can specify the detailed version for it.

Reply:
for example,  testing the upstream QEMU 8.0 rc build

Reply:
@yanan-fu Got it, thanks and updated.